### ISSUE TITLE
Selecting a color causes a JS-Error in Console that prevents using the color picker

### ISSUE THREAD

**Issue by NicoleNitschke** [Sebobo/Shel.Neos.ColorPicker#75]
I use the ColorPicker like described in the documentation, my yaml-Code is the following:

```
  properties:
    color:
      type: string
      defaultValue: 'grass'
      ui:
        label: i18n
        reloadIfChanged: true
        inspector:
          position: 0
          group: properties
          editor: 'Shel.Neos.ColorPicker/ColorPickerEditor'
          editorOptions:
            mode: 'preset'
            presetColors: [{color: '#d7ecdc', title: 'Grass', value: 'grass'}, {color: '#ccdcde', title: 'Sky', value: 'sky'}, {color: '#dfcec8', title: 'Rock', value: 'rock'}, {color: '#064349', title: 'Forest', value: 'forest'}]
```

In the Fusion Integration Part, I want to get my color with the following code:

`color = ${q(node).property('color')}`

When I open the Neos Backend now, I see the color picker with the defined default color. But when I try to change the color by clicking another color, nothing changes, but I get the following error in the console:

![grafik](https://user-images.githubusercontent.com/11499598/211544329-68e64b8e-1d21-47be-836a-75ad0fc7dc45.png)

**Comment by Sebobo** [comment]
Thx, can reproduce and fix is almost ready

**Comment by Sebobo** [comment]
Fix is in 1.6.3
